indico/htdocs/js/indico/angular/directives.js

```diff
@@ -468,3 +468,21 @@ ndDirectives.directive('ndInitFocus', function() {
         }, 0);
     };
 });
+
+ndDirectives.directive('ndJqueryDatepicker', function() {
+    return {
+        require: 'ngModel',
+        restrict: 'A',
+        link: function(scope, element) {
+            setTimeout(function() {
+                element.datepicker({
+                    dateFormat: scope.field.dateFormat,
+                    onSelect: function(date) {
+                        scope.date = date;
+                        scope.$apply();
+                    }
+                })
+            }, 0);
```

Comment:
you could use `_.defer(function() {...});` instead (also uses a 0-delay timeout, but better semantics in my opinion)